src/Workspaces/Core/Portable/PatternMatching/PatternMatcher.cs

```diff
@@ -170,15 +182,31 @@ private static bool ContainsUpperCaseLetter(string pattern)
             }
             else
             {
+                var isCaseSensitive = _compareInfo.IsPrefix(candidate, patternChunk.Text);
```

Comment:
`isPrefix`?

Reply:
no.  This is stating if it's case sensitive or not.  if it's a prefix that it is, otherwise it isn't and it must have been case inseisitive.